### ISSUE TITLE
Kubescape Scan doesn't work in Lens

### ISSUE THREAD

**Issue by yf-odyssey** [kubescape/kubescape#903]
Dear all，

I tested kubescape and it perform good! So I want to use the Lens to view several K8s cluster security report. But after I install lens-extension of kubescape，I click the Scan button and it doesn't work. It seemed that I met the similari problem as the https://github.com/kubescape/lens-extension/issues/3. Because I installed Lens in Windows enviroments , so I decided to open a new issue for it.
![198209847-910e44b8-c441-417d-8943-281b9977bd41](https://user-images.githubusercontent.com/33337184/200711291-54b451e4-0695-499e-9c5f-48f7ad14ad73.png)

The relevant information is described as follows:

I installed the Lens in windows 10 and the Lens is downloaded from it's website(https://k8slens.dev/). Lens version is the as below. But I think its version is v6.x because when lauch it the icon show that(Lens 6).
![198210275-176cd000-6356-4b30-8a54-fc375f6e70aa](https://user-images.githubusercontent.com/33337184/200711346-33f5ed8b-1a46-4d96-8a91-7cc1fa9dbe60.png)

I installed the K8s cluster in a centos (Centos 7)virtual machine and also installed the kubescape. The kubescape works correctly in CLI, just as below.
![198211299-ca277094-7812-42c4-8083-209a11320774](https://user-images.githubusercontent.com/33337184/200711394-58bc210c-f906-4f03-bf47-7b05c770e2e1.png)

I followed the instruction to instll kubescape extension in Lens .But it doesn't automatically scan the k8s cluster and click the scan button doesn't work also.
I read the "black screen black screen  #3" and also tried to uninstall the kubescape extension and deleted the files in the dir(**C:\Users\RKHD\AppData\Roaming\Lens\extension_data** and C:\Users\RKHD\AppData\Roaming\Lens\extension-store@kubescape). Then I installed it again but the problem remains. After I click the Scan button I checked the dir(C:\Users\RKHD\AppData\Roaming\Lens\extension_data\756ffa6a545a062276ff3de6d165494b811044faef8558f25d1f71baab813d8f) and find it's blank. Just as blow.
![198212746-2d7014fc-b878-49b1-997e-21fbc40453b8](https://user-images.githubusercontent.com/33337184/200711458-525409cb-32bd-43ca-8c66-b53447e53537.png)

Pls help me to find out the problem and thanks a lot!

**Comment by yf-odyssey** [comment]
I also uploaded the two json files in the dir(C:\Users\RKHD\AppData\Roaming\Lens\extension-store@kubescape\lens-extension), just as blow .I hope it will help.
![198223154-7ca5e312-1808-4c79-850f-e95e13cecbc7](https://user-images.githubusercontent.com/33337184/200711563-b3277ab5-09f9-4be9-a570-07abbfae0384.png)

https://github.com/kubescape/lens-extension/files/9876984/lens-extension.zip

I also check the two json files and find that the scanframework label is blank. Is it related to the problem? But I have downloaded the framwork manually just as below.
![198223352-67bbd593-4a5e-4fcb-ad00-e480f20ca68a](https://user-images.githubusercontent.com/33337184/200711693-07f77711-3e61-42b9-ae75-36dc547ff9f6.png)

**Comment by amirmalka** [comment]
@yf-odyssey  The issue you are describing could be a result of large output; If that is the case, it's an open issue we would fix in the future.
To confirm, please advise what is the size of the output file when you run kubescape as follows:
`kubescape scan --format json --format-version=v2 > output.json`

**Comment by dwertent** [comment]
@amirmalka I thought the Lens extension is running 'format-version=v1`

**Comment by yf-odyssey** [comment]
> @yf-odyssey The issue you are describing could be a result of large output; If that is the case, it's an open issue we would fix in the future. To confirm, please advise what is the size of the output file when you run kubescape as follows: `kubescape scan --format json --format-version=v2 > output.json`

Dear Amir，
 First thanks for your reply. I tried as you said and maybe the root cause is really related to the size of the output file. I run the command and find the output file is about 23.8M. The screenshot is as below.
<img width="638" alt="9" src="https://user-images.githubusercontent.com/33337184/201014192-01d2b891-4967-40cd-a370-9de61158b3ab.png">
![image](https://user-images.githubusercontent.com/33337184/201014264-847a4c8e-4e4a-4c1a-a806-6bd35916c924.png)

**Comment by amirmalka** [comment]
@yf-odyssey  @dwertent Our of habit I wrote `v2` - Lens extension still work `v1`.
Either way it satisfies the question as the v1 format could be around 3 times bigger.

As stated earlier, we hope to fix this soon - not just use the v2 format but also the way the extension reads the output, right now this is done from the standard output, which has its limits and not good enough.

**Comment by yf-odyssey** [comment]
Hi Amir, Thanks for your explanation. I hope this issue can be fixed ASAP. BTW,  I want to know if I install Lens in Mac or Linux and install the corresponding kubescape lens plugin, will the problem occurs?

**Comment by amirmalka** [comment]
@yf-odyssey Yes, the extension works the same across all operating systems. 
rest assured it will be fixed soon.

**Comment by yf-odyssey** [comment]
> @yf-odyssey Yes, the extension works the same across all operating systems. rest assured it will be fixed soon.

Ok, I got it. Thanks for your patient response. Then I'll wait the bug be fixed.
